docs/release.md

```diff
@@ -267,15 +267,12 @@ The release itself is only a git signed tag as it's used for minor releases as w
 
 # Merging to Release Branches
 
-> **Note:** Before a bug is fixed in a stable release branch, the same bug must
-> be fixed in the `master` branch. The only exception is when a bug exists in a
-> stable branch only.
-
 For every release a branch will be created following the pattern `release-x.y`.
 For now, community members can propose pull requests to be included into a
```

Comment:
Shouldn't we move lines 271-273 also to the new document?

Reply:
i thought those lines were covered in the new document? I give a brief explanation here and then link to the backporting doc where the same info is presented in more detail. 

is there something missing in the backporting doc that exists in the release doc?

Reply:
it's meant to be read as, bug fixes are okay to backport and feature enhancements are not.